### DISCUSSION TITLE
Repeat query by variable

### DISCUSSION THREAD

**Discussion by sandersaares** [grafana/grafana#61206] [Dashboards, Panels & Visualizations]
Some 3rd party data source plugins are not flexible enough to natively perform a query over many variable combinations. In these situations, it would help if I could define a "repeat query by variable" option, so the query is split into multiple queries, each of which the plugin can handle, and with the panel displaying the merged result (as if I had manually specified multiple queries).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). 

I'm having trouble understanding this feature request. Could you elaborate what what data source(s) youre most concerned with in an [issue](https://github.com/grafana/grafana/issues)? I'd like to better understand what you're looking for to continue the discussion.
